src/mongocxx/test/client_side_encryption.cpp

```diff
@@ -482,12 +500,13 @@ TEST_CASE("Datakey and double encryption", "[client_side_encryption]") {
 
 void run_external_key_vault_test(bool with_external_key_vault) {
     class client external_key_vault_client {
-        uri{"mongodb://fake-user:fake-pwd@localhost:27017"}, test_util::add_test_server_api()
+        uri{"mongodb://fake-user:fake-pwd@localhost:27017"},
+            test_util::add_test_server_api(crypt_shared_opts()),
```

Comment:
Remove `crypt_shared_opts()`.

The [External Key Vault Test](https://github.com/mongodb/specifications/tree/6018dda99f00d39056814dae0572a05bbd9b4868/source/client-side-encryption/tests#3-external-key-vault-test) does not require the external key vault client to be created with automatic encryption enabled.